src/app/components/CollapsableBookList.tsx

```diff
@@ -0,0 +1,56 @@
+import * as classNames from 'classnames';
+import { take } from 'lodash';
+import * as React from 'react';
+import MediaQuery from 'react-responsive';
+
+import { Icon } from '@ridi/rsg';
+import { ConnectedInlineHorizontalBookList } from 'app/components';
+import { Book } from 'app/services/book';
+import { Expander } from 'app/services/book/components/Expander';
+
+interface CollapsableBookListProps {
+  className: string;
+  listTitle: string;
+  uiPartTitleForTracking: string;
+  books: Book[];
+}
+
+export const CollapsableBookList: React.FunctionComponent<CollapsableBookListProps> = (props: CollapsableBookListProps) =>  {
```

Comment:
한 줄이 너무 길어지는 것 같아요...
```suggestion
export const CollapsableBookList: React.FunctionComponent<CollapsableBookListProps> = props => {
```

Reply:
FunctionComponent<> 에서 지정되어 뒷부분은 타입선언이 필요 없을까요?
그럼 빼는게 더 간결해보일 것 같네요.